docs/developer/add-new-source.md

```diff
@@ -1,6 +1,14 @@
 # How to add a new source to the Mondo Ingest
 
-1. Add a new import and component to [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
-2. Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
-3. Update the Website Table of Contents [mkdocs.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/mkdocs.yaml)
+## 1. `mondo-ingest-odk.yaml`
+### 1.1. Add source info
+Add a new filename and URL to the `products` > `components` section in [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
 
+### 1.2. `update_repo`
+From `src/ontology`, run `sh run.sh make update_repo`.
```

Comment:
@matentzn Added this next step about `update_repo`

Reply:
Yes, good!

Reply:
I added a note about it needing to run twice as well. However, as I mentioned in [this comment](https://github.com/monarch-initiative/mondo-ingest/issues/308#issuecomment-1550456717), running it a second time did not seem to do anything for me.